packages/design-system/vite.config.ts

```diff
@@ -1,42 +1,54 @@
-import { createVuePlugin } from 'vite-plugin-vue2';
+import vue from '@vitejs/plugin-vue2';
 import { resolve } from 'path';
+import { defineConfig, mergeConfig } from 'vite';
+import { defineConfig as defineVitestConfig } from 'vitest/config';
 
-export default {
-	plugins: [
-		createVuePlugin(),
-	],
-	resolve: {
-		alias: {
-			'@': resolve(__dirname, 'src'),
-			'vue2-boring-avatars': resolve(__dirname, '..', '..', 'node_modules', 'vue2-boring-avatars', 'dist', 'vue-2-boring-avatars.umd.js'),
-			// 'vue2-boring-avatars': 'vue2-boring-avatars/dist/vue-2-boring-avatars.umd.js',
-		},
-	},
-	build: {
-		lib: {
-			entry: resolve(__dirname, 'src', 'main.js'),
-			name: 'N8nDesignSystem',
-			fileName: (format) => `n8n-design-system.${format}.js`,
+export default mergeConfig(
+	defineConfig({
+		plugins: [
+			vue(),
+		],
+		resolve: {
+			alias: {
+				'@': resolve(__dirname, 'src'),
+				'vue2-boring-avatars': resolve(
+					__dirname,
+					'..',
+					'..',
+					'node_modules',
+					'vue2-boring-avatars',
+					'dist',
+					'vue-2-boring-avatars.umd.js',
+				),
+				// 'vue2-boring-avatars': 'vue2-boring-avatars/dist/vue-2-boring-avatars.umd.js',
```

Comment:
i think you could replace this with
```suggestion
				'vue2-boring-avatars': require.resolve('vue2-boring-avatars'),
```